mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,75 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
+
+  /**
+   * Enlarge Image Styles
+   */
+  isDisableZoom?: boolean;
```

Comment:
what the difference between isZoomable and isDisableZoom? I whink we should get rid of one of them

Reply:
No difference, removed isDisableZoom